meson.build

```diff
@@ -1529,8 +1524,8 @@ if features['cocoa'] and features['swift']
 endif
 
 macos_cocoa_cb = get_option('macos-cocoa-cb').require(
-    features['cocoa'] and features['swift'],
-    error_message: 'Either cocoa or swift could not be found!',
+    features['cocoa'] and features['gl-cocoa'] and features['swift'],
+    error_message: 'Either cocoa, gl-cocoa or swift could not be found!',
```

Comment:
Does it make sense to require `gl-cocoa` here? The metal layer doesn't require gl right? Maybe `video/out/mac/gl_layer.swift` should be pulled out of the `'macos-cocoa-cb` check and into a separate one.